components/cdc/src/initializer.rs

```diff
@@ -197,24 +212,16 @@ impl<E: KvEngine> Initializer<E> {
             old_value_cursors = Some(OldValueCursors::new(wc, dc));
         }
 
-        // Time range: (checkpoint_ts, max]
-        let mut scanner = ScannerBuilder::new(snap, TimeStamp::max())
-            .fill_cache(false)
-            .range(None, None)
-            .hint_min_ts(hint_min_ts)
-            .build_delta_scanner(self.checkpoint_ts, TxnExtraOp::ReadOldValue)
-            .unwrap();
-
-        fail_point!("cdc_incremental_scan_start");
-        let conn_id = self.conn_id;
-        let mut done = false;
-        let start = Instant::now_coarse();
-
         let curr_state = self.downstream_state.load();
         assert!(matches!(
             curr_state,
             DownstreamState::Initializing | DownstreamState::Stopped
         ));
+
+        fail_point!("cdc_incremental_scan_start");
```

Comment:
Why reorder code?

Reply:
Sorry, I didn't notice the order here. I have restored here.

Reply:
Sorry, I didn't notice the order here. I have restored t